src/test/java/org/broadinstitute/consent/http/MockServerTestHelper.java

```diff
@@ -0,0 +1,38 @@
+package org.broadinstitute.consent.http;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.platform.launcher.TestExecutionListener;
+import org.junit.platform.launcher.TestPlan;
+import org.mockserver.client.MockServerClient;
+import org.testcontainers.containers.MockServerContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+public class MockServerTestHelper extends AbstractTestHelper implements TestExecutionListener,
+    WithMockServer {
+
+  public static final MockServerContainer container = new MockServerContainer(IMAGE).waitingFor(
+      Wait.forLogMessage(".*started on port:.*", 1));
+  public static MockServerClient mockServerClient;
+
+  static void startUp() {
```

Comment:
Does this need to be a separate method? its only use is in this file.
```suggestion
  private static void startUp() {
```

Reply:
Refactor coming to eliminate the method.